<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="fromLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="subjectLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="contentBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>431</width>
     <height>441</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Od:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>42</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Temat:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>42</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>42</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Treść:</string>
   </property>
  </widget>
  <widget class="QLabel" name="attachmentLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>620</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Załączniki:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dowloadButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pobierz załączniki</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
